Automattic/wp-calypso
# Performance Profiler: Add link to recommendations on CWW section

Fixes [Performance Profiler: add link to jump to recommendations](https://github.com/Automattic/dotcom-forge/issues/9048)

## Proposed Changes

Add a link to the recommendations section on the CWW section.
The link should redirect to the section with using the current metric as filter.

![CleanShot 2024-09-11 at 14 27 38](https://github.com/user-attachments/assets/c4981662-de4e-4906-be07-a76c59d26a0b)


PS: If there are no recommendations for a given metric, this section shouldn't be shown.

## Why are these changes being made?

[Performance Profiler: add link to jump to recommendations](https://github.com/Automattic/dotcom-forge/issues/9048)


## Testing Instructions

* Go to `speed-test-tool?url=https%3A%2F%2Fwordpress.com%2F&hash=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpZCI6NzQ2N30.LOelpUBxFDxMytpSYeB3EBNdBnHeYTvonyZ6zE7aqSM`
* On the CWW section check if there is a button with `View recommendations`
* Click on this button 
* Check if you the page scroll to the recommendations section with the selected metric on the filter.
* Check if the section is not shown for some metrics
  * It shouldn't be shown where are no recommendations for a given metric.
